src/Validation.ContentScan.Core/ContentScanStatusMessage.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Validation.ContentScan.Core
+{
+    public class ContentScanStatusMessage
+    {
+        public ContentScanStatusMessage(
+           Guid packageValidationId)
+        {
+            PackageValidationId = packageValidationId;
+        }
+        public Guid PackageValidationId { get; }
```

Comment:
Please add an empty line to separate constructor and property:

```suggestion

        public Guid PackageValidationId { get; }
```